### DISCUSSION TITLE
ABP google indexed url are failing with 404. it will be great if there is a redirect from old url to new url.

### DISCUSSION THREAD

**Discussion by antosubash** [abpframework/abp#20286] [General]
There are lots of ABP documentation pages which are indexed by the google are going to the 404 page. it is same for the internal documentation we had. It seems like most of the old abp url are broken. and there is no global search. finding info is bit hard right now. I usally use google for searching abp docs but it seems like it is not possible until google reindex all the pages again.

**Comment by maliming** [comment]
hi

Can you share some 404 urls?

**Comment by antosubash** [comment]
https://www.google.com/search?q=abp+multitenancy&rlz=1C1FHFK_enAT1112AT1112&oq=abp+mu&gs_lcrp=EgZjaHJvbWUqDggAEEUYJxg7GIAEGIoFMg4IABBFGCcYOxiABBiKBTIGCAEQRRg5MgcIAhAAGIAEMgcIAxAAGIAEMgcIBBAAGIAEMgYIBRBFGDwyBggGEEUYPDIGCAcQRRg8qAIAsAIA&sourceid=chrome&ie=UTF-8



https://docs.abp.io/cs/abp/latest/Multi-Tenancy

This is the old url.

https://abp.io/docs/latest/framework/architecture/multi-tenancy#gsc.tab=0

This is the new url.

**Reply by maliming** [reply]
Thanks. We will fix the 404 problem.
